interfaces.go

```diff
@@ -15,15 +15,25 @@ type Connection interface {
 
 // KeySpace is used to obtain tables from.
 type KeySpace interface {
-	MapTable(tableName, id string, row interface{}) MapTable
-	MultimapTable(tableName, fieldToIndexBy, uniqueKey string, row interface{}) MultimapTable
-	MultimapMultiKeyTable(tableName string, fieldToIndexBy, uniqueKey []string, row interface{}) MultimapMkTable
-	TimeSeriesTable(tableName, timeField, uniqueKey string, bucketSize time.Duration, row interface{}) TimeSeriesTable
-	MultiTimeSeriesTable(tableName, fieldToIndexByField, timeField, uniqueKey string, bucketSize time.Duration, row interface{}) MultiTimeSeriesTable
-	MultiKeyTimeSeriesTable(tableName string, fieldToIndexByField []string, timeField string, uniqueKey []string, bucketSize time.Duration, row interface{}) MultiKeyTimeSeriesTable
-	FlakeSeriesTable(tableName, idField string, bucketSize time.Duration, row interface{}) FlakeSeriesTable
-	MultiFlakeSeriesTable(tableName, indexField, idField string, bucketSize time.Duration, row interface{}) MultiFlakeSeriesTable
-	Table(tableName string, row interface{}, keys Keys) Table
+	// MapTable is a simple key-value store.
+	MapTable(prefixForTableName, partitionKey string, row interface{}) MapTable
```

Comment:
I wonder if we can improve upon `row`, maybe something like `obj`, `definition`, `fieldDefinition`?

Reply:
I thought row is the "official" Cassandra term for a key/value "pair"? I do agree `row` doesn't mean much though. Maybe `rowDefinition` 🤔

Reply:
I think `rowDefinition` makes sense and provides clarity :)